LayoutTests/fast/forms/time/time-stepup-stepdown-expected.txt

```diff
@@ -24,14 +24,15 @@ Step=any
 PASS stepUp("20:13", "any", null) threw exception InvalidStateError: The object is in an invalid state..
 PASS stepDown("20:13", "any", null) threw exception InvalidStateError: The object is in an invalid state..
 Overflow/underflow
-PASS stepUp("20:13", "3.40282346e+38", null) is "20:13"
-PASS stepDown("20:13", "3.40282346e+38", null) is "00:00:00"
+FAIL stepUp("20:13", "3.40282346e+38", null) should be 20:13. Was 00:00:00.
+FAIL stepDown("20:13", "3.40282346e+38", null) should be 20:13. Was 00:00:00.
```

Comment:
Does `time-stepup-stepdown.html` need to be updated further?

Reply:
These fail on Blink upstream as well.

Reply:
Updated based on our discussion and in summary - similar to other input types, they should reset and we just need to fix test.